doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -229,6 +229,27 @@ Securing Solr
 
 Our sample init script and systemd service file linked above tell Solr to only listen on localhost (127.0.0.1). We strongly recommend that you also use a firewall to block access to the Solr port (8983) from outside networks, for added redundancy.
 
+
+To start Solr Apache in external use mode, you must access the Solr example directory:
+
+# cd /usr/local/solr-7.7.2/bin/
+
+# ./solr start -j "-Djetty.host = DNS” -force
+
+Right after that you should use the iptables blocking rule
+
+# iptables -A INPUT -s ip address / 32 -p tcp -m tcp --dport 8983 -j ACCEPT
+
+# iptables -A INPUT -p tcp -m tcp --dport 8983 -j DROP
```

Comment:
For all of these commands please use the format for "usermod" and "sudo" in the screenshot below. This ensures consistency and prevents quotes from being turning into smart quotes.

<img width="976" alt="Screen Shot 2021-04-08 at 11 15 34 AM" src="https://user-images.githubusercontent.com/21006/114052367-fe403a00-985b-11eb-903c-fd3fe657bf47.png">

Reply:
I agree, not least because it validates that the execution will be correctly. use command sudo.